### PR TITLE
SPA SDK

### PR DESCRIPTION
**Pull request by erjohnson**
* SPA SDK docs under Libraries.
* Updated sidebar navigation

### CODE REVIEW

File: articles/libraries/auth0-spa-js/index.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+topics:
+  - libraries
+  - auth0-spa-js
+contentType:
+  - index
+---
+
+# Auth0 SDK for Single Page Applications
+
+Auth0 SDK for Single Page Applications using the [Authorization Code Grant Flow with PKCE](/api-auth/tutorials/authorization-code-grant-pkce).
```
Comment (erjohnson) [inline articles/libraries/auth0-spa-js/index.md]: needs better intro

File: articles/libraries/auth0-spa-js/index.md [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+You can find the source code of the library [on Github](https://github.com/auth0/auth0-spa-js) and the full API documentation is [here](https://auth0.github.io/auth0-spa-js/).
+
+## Installation
+
+You have a few options for using auth0-spa-js in your project:
+
+From the CDN:
+
+```html
+<script src="https://cdn.auth0.com/js/auth0-spa-js/1.0.0-beta.2/auth0-spa-js.production.js"></script>
```
Comment (luisrudge) [inline articles/libraries/auth0-spa-js/index.md]: let's use 1.0.0 from the get go, so we don't forget about it later 😬

**Review by luisrudge — APPROVED** [review]

File: articles/libraries/auth0-spa-js/index.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+```
+
+```js
+//with async/await
+document.getElementById('callApi').addEventListener('click', async () => {
+  const accessToken = await auth0.getTokenSilently();
+  const result = await fetch('https://exampleco.com/api', {
+    method: 'GET',
+    headers: {
+      Authorization: `Bearer ACCESS_TOKEN`
```
Comment (luisrudge) [inline articles/libraries/auth0-spa-js/index.md]:
```suggestion
      Authorization: `Bearer ${accessToken}`
```

Comment (luisrudge) [inline articles/libraries/auth0-spa-js/index.md]:
```suggestion
      Authorization: `Bearer ${accessToken}`
```

File: articles/libraries/auth0-spa-js/index.md [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+
+//with promises
+document.getElementById('callApi').addEventListener('click', () => {
+  auth0
+    .getTokenSilently()
+    .then(accessToken =>
+      fetch('https://myapi.com', {
+        method: 'GET',
+        headers: {
+          Authorization: `Bearer ACCESS_TOKEN`
```
Comment (luisrudge) [inline articles/libraries/auth0-spa-js/index.md]:
```suggestion
      Authorization: `Bearer ${accessToken}`
```

File: articles/libraries/auth0-spa-js/index.md [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+
+```js
+document.getElementById('logout').addEventListener('click', () => {
+  auth0.logout();
+});
+```
+
+## Usage
+
+JQuery is used in the following examples.
```
Comment (luisrudge) [inline articles/libraries/auth0-spa-js/index.md]:
```suggestion
jQuery is used in the following examples.
```

File: articles/libraries/auth0-spa-js/index.md [hunk 6]
```diff
@@ -0,0 +1,287 @@
...
+### Login with Popup
+
+```html
+<button id="loginPopup">Login with Popup</button>
+```
+
+```js
+$('#loginPopup').click(async () => {
+  await auth0.loginWithPopup({
+    redirect_uri: 'http://localhost:3000/callback.html'
```
Comment (luisrudge) [inline articles/libraries/auth0-spa-js/index.md]: there's no redirect_uri in `loginWithPopup `

**Review by luisrudge — CHANGES_REQUESTED** [review]

File: articles/libraries/auth0-spa-js/index.md (hunk 3 above)
Comment (erjohnson) [reply articles/libraries/auth0-spa-js/index.md]: using {$accessToken} gives us a build error

**Review by luisrudge — APPROVED** [review]

**Pull request merged**